BLSTM.lua

```diff
@@ -0,0 +1,67 @@
+local BLSTM, parent = torch.class('cudnn.BLSTM', 'cudnn.RNN')
+local errcheck = cudnn.errcheck
+
+function BLSTM:__init(inputSize, hiddenSize, numLayers)
```

Comment:
Does it make sense to add self.numDirections to base RNN class (set it to 1 for uni-directional, and 2 for bi-directional), and use this var to avoid duplicating all of __init, resizeOutput, resizeHidden and resetIODescriptors?
That way, for BLSM you will just have to call parents __init, and set self.numDirections, self.bidirectional and self.mode in __init. resizeOutput, resizeHidden and resetIODescriptors can be reused for bidirectional nets w/o changes.